oops/backplane/unittester_support.py

```diff
@@ -0,0 +1,503 @@
+################################################################################
+# oops/backplane/unittester_support.py
+################################################################################
+
+import unittest
+import re
+import os
+import sys
+import numpy as np
+
+from polymath   import Scalar
+from PIL        import Image
+
+
+#===============================================================================
+def _read_image(filename):
+    """
+    Save an image file of a 2-D array.
+
+    Input:        filename    the name of the input file, which should end with
+                              the type, e.g., '.png' or '.jpg'
+    """
+    try:
+        im = Image.open(filename, mode="r")
+    except:
+        return None
+
+    return Scalar.as_scalar((np.array(im)))
+
+
```

Comment:
Mark should chime in here, but in general it seems like the amount of vertical whitespace is getting excessive everywhere. One of our coding style guidelines is to make as much fit on the screen vertically at a time as possible. I don't think having 3 or 4 blank lines PLUS a divider is necessary for readability.